### ISSUE TITLE
[suggestion] Add a constant combinator into the train stop

### ISSUE THREAD

**Issue by BloodyRain2k** [Yousei9/Logistic-Train-Network#301]
**TL;DR**
Add a constant combinator into the train stop for it's settings so there's no need to place an extra one nearby for the static signals.

**Why ?**
Given that there's already a lamp, and whatever the output entity was, squeezed into the 2x2 of the train stop, I was thinking that it would be nice if there could be also a constant combinator added for setting the static signals of the stop.

There's a mod that adds a 'special' constant combinator which makes using LTN a LOT easier to understand (and remember how it worked) for people who have not spend hundreds of hours with setups.
https://mods.factorio.com/mod/LTN_Combinator_Modernized

Having the built-in combinator use that layout would be a nice addition, maybe also with tooltips for the different settings so they're even better to understand how they work. The stack limits for example.

**Comment by Yousei9** [comment]
LTN Combinator exists as its own optional mod, by other developers.
No need to add it into LTN itself.
